### PR TITLE
Updating mobile SDK doc to 0.0.2

### PR DESCRIPTION
**Pull request by fan-zhang-sv**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: docs/pages/guides/react-native-integration.mdx [hunk 1]
```diff
@@ -106,36 +127,114 @@ useEffect(() => {
 ```
 
 ## Usage
+
+Mobile Wallet Protocol Client provides 2 interfaces for mobile app to interact with the Smart Wallet, an EIP-1193 compliant provider interface and a wagmi connector.
+
+<Callout type="success" >
+If your app is using wallet aggregator, go straight to **Option 2: Wagmi Connector** for 1-line integration. :)
```
Comment (vishnumad) [inline docs/pages/guides/react-native-integration.mdx:134]: Nit: Might be nice to link to the Option 2 fragment here: `#option-2-wagmi-connector`

**Review by vishnumad — APPROVED** [review]
LGTM, left a nit

**Comment by cb-heimdall** [comment]
Review Error for vishnumad @ 2024-08-15 20:11:02 UTC
User must have write permissions to review

**Review by wilsoncusack — APPROVED** [review]

**Pull request merged**